### PR TITLE
Add community section to home page.

### PR DESCRIPTION
**Pull request by chexxor**
Requested in https://github.com/purescript/purescript.github.io/issues/125

I didn't test this locally yet. Wanted to get a PR out first.

### CODE REVIEW

**Comment by soupi** [comment]
Hey I [just made the same changes](https://github.com/soupi/purescript.github.io/commit/fd17601c8c9cc23fce99f1e493e838d08cfb48b4) :+1: 

I decided to keep only the ones i thought were really important (irc, discourse, gatherings) and wrote less about them, but you are in a better position than me to decide what you want to advertise.

**Comment by chexxor** [comment]
Here's a screenshot of my patch, to make it easier to review.

image

@soupi Yeah, as I have it in this screenshot, it's pretty hard to read. I'll make another commit to try to simplify it.

**Comment by paf31** [comment]
Thanks!

- I would remove the reference to Google Groups. It's been dead for ages.
- Also there is a typo in `discource`.
- I don't think we need to reference Twitter, since people presumably know how to search there if they use it.
- Generally, I'd like to reduce the wordiness of this a bit, and use names like `FP Slack` instead of raw URLs in the copy.

**Comment by chexxor** [comment]
Here's a screenshot of the latest commit:
image

**Comment by soupi** [comment]
My commit looks like this, if you like i can make a PR.

**Comment by chexxor** [comment]
My latest commit, inspired by the beautiful conciseness of @soupi's.
image

File: index.html [hunk 1]
```diff
@@ -216,10 +216,10 @@ <h3>Book</h3>
...
-            <li><strong>IRC</strong> -- The <a href="irc://irc.freenode.net/purescript">#purescript channel on IRC Freenode</a>. It is slightly less active than the Slack channel above.</li>
-            <li><strong>Discourse</strong> -- The <a href="https://discourse.purescript.org">PureScript Discourse</a> has become a popular web forum.</li>
-            <li><strong>Regular online gatherings</strong> -- Individual events can be found and  RSVPed at <a href="https://gathering.purescript.org/">gathering.purescript.org</a>, while the schedule is documented in the <a href="https://github.com/purescript/documentation/blob/master/ecosystem/PureScript-Meetups.md">github.com/purescript/documentation project</a>.</li>
+            <li><strong>Slack</strong> -- The #purescript channel on <a href="https://functionalprogramming.slack.com/">FP Slack</a>. Use the <a href="https://fpchat-invite.herokuapp.com/">FP Slack invite bot</a> to make an account there.</li>
```
Comment (paf31) [inline index.html]: Can you please replace the double dashes with single dashes or em-dashes.

File: index.html [hunk 2]
```diff
@@ -216,10 +216,10 @@ <h3>Book</h3>
...
+            <li><strong>IRC</strong> -- The <a href="irc://irc.freenode.net/purescript">#purescript channel on IRC Freenode</a>.</li>
```
Comment (paf31) [inline index.html]: s/IRC Freenode/Freenode please

File: index.html [hunk 3]
```diff
@@ -216,10 +216,10 @@ <h3>Book</h3>
...
+            <li><strong>Discourse</strong> -- The <a href="https://discourse.purescript.org">PureScript Discourse</a> instance.</li>
+            <li><strong>Regular online gatherings</strong> -- Individual events can be found and RSVPed at <a href="https://gathering.purescript.org/">gathering.purescript.org</a>, while the schedule is documented in the <a href="https://github.com/purescript/documentation/blob/master/ecosystem/PureScript-Meetups.md">github.com/purescript/documentation project</a>.</li>
```
Comment (paf31) [inline index.html]: Sorry, can you please remove "and RSVPed" here

Comment (paf31) [reply index.html]: Also can you please cut this down a bit further by just saying something like "and the schedule is documented <link>here</link>"

**Comment by chexxor** [comment]
Nice ideas! It's much better after making those changes.

image

**Comment by paf31** [comment]
Thanks!

**Pull request merged**